libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -1050,6 +1052,7 @@ class MapboxNavigation @VisibleForTesting internal constructor(
             ReachabilityService.removeReachabilityObserver(it)
             reachabilityObserverId = null
         }
+        routeRefreshController.observer = null
```

Comment:
The AS says it's experimental but it's invoked from the method that does not opt-in. Is it a standard practice?

Reply:
it's hidden logic, which is invoked under the hood. Otherrwise we must put experimental (optIn) annotation to `onDestroy` method, that I would avoid to do

Reply:
Yes, I get it, it's just that the chain is broken. But I don't have a solution so let's leave as-is.